docs/preview/02-Features/assertion.md

```diff
@@ -0,0 +1,210 @@
+
+💡 It provides you with more options to control how your file should be loaded.
+
+```csharp
+using System.Xml;
+using Arcus.Testing;
+
+string xml = ...;
+
+XmlDocument expected = AssertXml.Load(json);
+
+// Or with options (same as available with `AssertJson.Equal`).
+XmlDocument actual = AssertXml.Load(json, options => ...);
+
+// Use overload with nodes instead.
+AssertXml.Equal(expected, actual);
+```
+
+> ❓ Why use the `AssertXml` to load something that is already available with `XmlDocument.LoadXml`?
+
+The `AssertXml.Load` is a special variant on the existing load functionality in such a way that it provides more descriptive information on the input file that was trying to be parsed, plus by throwing an assertion message, it makes the test output more clear on what the problem was and where it happened. 
+
+## JSON
+The library has an `AssertJson` class that exposes useful test assertions when dealing with JSON outputs. The most popular one is comparing JSON documents.
+
+```csharp
+using Arcus.Testing;
+
+string expected = "{ \"root\": ... }";
+string actual = "{ \"diff-root\": ... }";
+
+AssertJson.Equal(expected, actual);
+// Arcus.Testing.EqualAssertionException
+// AssertJson.Equal failure: expected and actual JSON contents do not match
+// Actual JSON misses property at $.root
+//
+// Expected:
+// {
+//    "root": ...
+// }
+//
+// Actual:
+// {
+//    "diff-root": ...
+// }
+```
+
+💡 Currently, the input contents are trimmed in case if the input is too big to be shown in a humanly readable manner to the test output. In case of large files, it might be best to log those files (or parts that interest you) separately before using this test assertion.
+
+### Customization
+The test assertion also expose several options to tweak the behavior of the JSON comparison.
+
+```csharp
+using Arcus.Testing;
+
+AssertJson.Equal(..., options =>
+{
+    // Adds one ore more property names of JSON nodes that should be excluded from the JSON comparison.
+    options.IgnoreNode("node-name");
+
+    // Sets the type of order which should be used when comparing JSON array values.
+    // REMARK: only the order of JSON values can be set, JSON objects within JSON arrays are still compared by their contents.
+    // Default: Ignore.
+    options.Order = AssertJsonOrder.Include;
+
+    // Sets the maximum characters of the expected and actual inputs should be written to the test output.
+    // Default: 500 characters.
+    options.MaxInputCharacters = 1000;
+});
+```
+
+### Loading JSON nodes yourself
+The JSON assertion equalization can be called directly with raw contents - internally it parses to a valid JSON structure: `JsonNode`. It it so happens that you want to compare two JSON nodes with different serialization settings, you can load the two nodes separately and do the equalization on the loaded nodes.
```

Comment:
Same as above for second sentence:
Second sentence not clear: "It it so happens that"
Maybe change by "If you want to compare..."

Reply:
Yes, fixed that here: https://github.com/arcus-azure/arcus.testing/pull/110/commits/ae05a045b4b05a1bad21167bccc162bddd1319f7